test/account/runner.go

```diff
@@ -64,16 +64,17 @@ func createAccountClientsFromManifest(t *testing.T, fs afero.Fs, manifestFileNam
 	return accClients
 }
 
-// randomizeYAMLResources loops over each *.yaml file, replaces %RAND% with a random string and returns the random string
+// randomizeAndResolveYAMLResources loops over each *.yaml file, replaces %RAND% with a random string, resolves environment variables set via ${MY_ENV} and returns the random string
```

Comment:
I do not like that we have multiple different methods of defining those testing replacements, please unify them